docs/en/core-services/Catalog.md

```diff
@@ -213,7 +213,13 @@ and additions in the UDB tables.
 
 ## Using Alluxio Structured Data with Presto
 
-The Alluxio Catalog Service is built to be used with the connector in Presto.
+PrestoSQL version above 332 and PrestoDB version above 0.232 has built-in support for the Alluxio Catalog Service in their hive-hadoop2 connector.
```

Comment:
I think this should be worded by `332 and above` or `333 and above` depending on which is true. This applies to `0.232` as well.